docs/source/author/chapters.rst

```diff
@@ -52,11 +50,9 @@ Configuration Files
 ==============================================================================
```

Comment:
The above paragraphs need to be changed to reflect what I talked about before. it's mostly fine, which confuses me because some of it contradicts the stuff I had problems with earlier